src/components/ChatComponents/ChatControls.tsx

```diff
@@ -76,13 +75,6 @@ const ChatIcons: React.FC<ChatIconsProps> = ({
         if (vault_qa_strategy === VAULT_VECTOR_STORE_STRATEGY.ON_MODE_SWITCH) {
           await onRefreshVaultContext();
         }
-        const activeNoteOnMessage: ChatMessage = {
-          sender: AI_SENDER,
-          message: `OK Feel free to ask me questions about your vault: **${app.vault.getName()}**. \n\nIf you have *NEVER* as your auto-index strategy, you must click the *Refresh Index* button below, or run Copilot command: *Index vault for QA* first before you proceed!\n\nPlease note that this is a retrieval-based QA. Specific questions are encouraged. For generic questions like 'give me a summary', 'brainstorm based on the content', Chat mode with direct \`[[note title]]\` mention is a more suitable choice.`,
```

Comment:
This message below is worth keeping above the "NEVER" warning unconditionally for Vault QA. 

"Please note that this is a retrieval-based QA. Questions should contain keywords and concepts that exist literally in your vault."

Reply:
Updated
<img width="489" alt="Screenshot 2024-11-02 at 11 01 01 PM" src="https://github.com/user-attachments/assets/2e3749c2-665f-47d1-8b83-61f032dbbae3">
<img width="491" alt="Screenshot 2024-11-02 at 11 01 28 PM" src="https://github.com/user-attachments/assets/ebb2d1cd-246c-459c-90b2-8703aac4df74">